### PR TITLE
Sanitize

### PR DESCRIPTION
**Pull request by Alan-Cha**

### CODE REVIEW

File: packages/openapi-to-graphql/src/index.ts [hunk 1]
```diff
@@ -539,3 +539,5 @@ function preliminaryChecks(
     })
   }
 }
+
+export { sanitize } from './oas_3_tools'
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/index.ts]: Users probably want this, right?

File: packages/openapi-to-graphql/src/oas_3_tools.ts [hunk 2]
```diff
@@ -466,7 +468,7 @@ export function inferResourceNameFromPath(path: string): string {
     }
   })
 
-  return sanitize(pathNoPathParams)
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/oas_3_tools.ts]: We run `sanitize` in `createDataDef`. Before then, all potential names are not sanitized, so sanitizing here (early) seems to be redundant.

File: packages/openapi-to-graphql/src/oas_3_tools.ts [hunk 3]
```diff
@@ -493,19 +495,19 @@ export function getRequestSchema(
...
-      // Prioritizes content-type JSON
+      // Prioritize content-type JSON
       if (Object.keys(content).includes('application/json')) {
         return {
           payloadContentType: 'application/json',
           payloadSchema: content['application/json'].schema as SchemaObject
         }
       } else {
-        // Picks a random content type
-        for (let contentType in content) {
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/oas_3_tools.ts]: I think this code might be confusing/bad

File: packages/openapi-to-graphql/src/oas_3_tools.ts [hunk 4]
```diff
@@ -624,19 +625,19 @@ export function getResponseSchema(
...
-        // Prioritizes content-type JSON
+        // Prioritize content-type JSON
         if (Object.keys(content).includes('application/json')) {
           return {
             responseContentType: 'application/json',
             responseSchema: content['application/json'].schema as SchemaObject
           }
         } else {
-          // Picks a random content type
-          for (let contentType in content) {
```
Comment (Alan-Cha) [inline packages/openapi-to-graphql/src/oas_3_tools.ts]: Similar change as above.

**Comment by ErikWittern** [comment]
@Alan-Cha Beautiful and sane!

**Pull request merged**